addons/binding/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/TVControlChannel.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lgwebos.internal;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.lgwebos.handler.LGWebOSHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.connectsdk.core.ChannelInfo;
+import com.connectsdk.device.ConnectableDevice;
+import com.connectsdk.service.capability.TVControl;
+import com.connectsdk.service.capability.TVControl.ChannelListener;
+import com.connectsdk.service.command.ServiceCommandError;
+import com.connectsdk.service.command.ServiceSubscription;
+import com.google.common.collect.Iterables;
```

Comment:
Do we really need this (dependency) imho the Java 8 way is more standard and should be prefered can you take a quick look how hard it is to switch to the normal Java 8 of doing such things?

Reply:
Well this stems from earlier versions when java 8 was not the standard in OH yet, but sure.. Multiple occurrences replaced.

Reply:
and now got rid of both google commons dependencies:
```
- com.google.common.base,
- com.google.common.collect,
```

Reply:
👍